src/services/user.service.js

```diff
@@ -0,0 +1,33 @@
+const { User } = require('../models/User.model.js');
+
+const getAllUsers = async () => {
+  return User.findAll();
+};
+
+const getUserById = async (id) => {
+  return User.findByPk(id);
+};
+
+const createUser = async (name) => {
+  return User.create({ name });
+};
+
+const updateUser = async ({ id, name }) => {
+  await User.update({ name }, { where: { id } });
```

Comment:
Ensure that the update operation handles cases where the provided name might be undefined. You might want to validate the input before updating to avoid overwriting existing data with undefined values.